win32ss/user/user32/controls/button.c

```diff
@@ -1351,7 +1332,7 @@ static void OB_Paint( HWND hwnd, HDC hDC, UINT action )
     dis.itemData   = 0;
     GetClientRect( hwnd, &dis.rcItem );
 
-    if ((hFont = get_button_font( hwnd ))) hPrevFont = SelectObject( hDC, hFont );
+    if ((hFont = get_button_font( hwnd ))) SelectObject( hDC, hFont );
```

Comment:
Isn't there going to be a GDI object leak once the OB_Paint function returns? (since there's no SelectObject to restore the original font)

Reply:
https://gitlab.winehq.org/wine/wine/-/commit/b7a08c92811c0a6c10b65b8b317e7f2d178e5bf1

it kinda looks like this was causing a test to fail to me.